packages/core/utils/src/modules-sdk/joiner-config-builder.ts

```diff
@@ -79,16 +79,22 @@ export function defineJoinerConfig(
             pluralize(upperCaseFirst(alias.args.entity)),
         },
       })),
-      ...models.map((entity, i) => ({
-        name: [
-          `${camelToSnakeCase(entity.name).toLowerCase()}`,
-          `${pluralize(camelToSnakeCase(entity.name).toLowerCase())}`,
-        ],
-        args: {
-          entity: entity.name,
-          methodSuffix: pluralize(upperCaseFirst(entity.name)),
-        },
-      })),
+      ...models
+        .filter((model) => {
+          return !alias
```

Comment:
It's a bit strange to have a tertiary check in a filter, and it's difficult to understand what it does. Maybe we can rewrite it to
```
!alias || !alias.some(...)
```